securedrop/journalist.py

```diff
@@ -195,8 +195,13 @@ def admin_add_user():
                     flash("That username is already in use",
                           "error")
                 else:
-                    flash("An error occurred saving this user to the database",
+                    flash("An error occurred "
+                          "saving this user to the database. "
+                          "Please check the application "
+                          "logs or inform your adminstrator.",
                           "error")
+                    app.logger.error("Adding user '{}' failed: {}".format(
```

Comment:
I don't think you should make this change here, but generally speaking it might be good to consider logging tracebacks instead of just exception messages in the application logs. @freedomofpress/securedrop-maintainers